tests/x11/libqt5_qtbase.pm

```diff
@@ -30,26 +31,25 @@ sub run {
 
     select_console('x11');
     ensure_installed("libqt5-qttools yast2-installation", timeout => 180);
+    # performance issue (see poo#125873) impacts only aarch64 workers on OSD
+    designer_qt5() unless (is_aarch64 && is_sle);
+    release_notes() unless (is_aarch64 && is_sle);
```

Comment:
Did we miss any test coverage on aarch64 platform with this change?

Reply:
Yes of course, the whole qt5 designer path is omitted.

If possible I would keep the `release_notes()` test. For the QT5 designer part .... I see that's a painful test run and the number of people who run qt5designer on aarch64 is probably rather low.

Let's keep the `release_notes()` though?

Reply:
who wants do this on aarch64 and s390x in real life?
release_notes is not relevant to openSUSE anyway, but you know I don't want drop this as long as it works...
by the way, just start and close application is too less for function test actually, but better than nothing.

Reply:
It's a smoke test for the qt bindings in YaST as such I would keep them, unless you think they are a major pain. I leave the decision to you, the PR is IMHO good as it is.